### DISCUSSION TITLE
افت سرعت شدید بعد از نصب حتی با پایین بودن پینگ

### DISCUSSION THREAD

**Discussion by MirzaT1992** [Gozargah/Marzban#464] [General]
سلام
من پنل و نصب میکنم دامین رو میدم ولی هم سرور برا خودم بد باز میشه هم سرعت اکانت خیلی پایینه
ولی پینگ روی ۵۰۰ هست
ممنون میشم راهنمایی کنید

**Comment by ituser96** [comment]
با نتهای مختلف چک کن. مثلا هر کجا نت داری از جای دیگه.. شاتل،مخابرات،ایرانسل،همراه اول،رایتل و....
 ممکنه با نت خودت اینطوری باشه. و اینکه کلا سعی کن قبل اینکه دامنه بدی و سایت های ایرانی رو محدود کنی اول تست رو بگیری بعد اوکی بود یکی یکی برو مرحله بعدی..

**Reply by MirzaT1992** [reply]
ممنون از وقتی که گذاشتی جالب همینه وقتی نصب میکنم هم یبار میاد یبار نمیاد هست احتمال داره ای پی رو محدود کردن یا دامین من محدود شده؟

**Comment by ituser96** [comment]
احتمال زیاد از آی پی باشه؛ چون من خودمم بار اول از فروشنده ایرانی خریدم و فقط میتونم بگم بهم انداخت؛ اونم سرویسی که آی پی بلاک بوده؛
 بار بعدی مستقیم از خارج گرفتم هنوزم داره عالی جواب میده. ایرانیا اول پولو میگیرن بعد خودشون لو میدن که نتونی از منابعشون استفاده کنی. نامردن.
